### PR TITLE
Ooyala api hook

### PR DESCRIPTION
**Pull request by sgupta3112**
Hi @rcvrgs , 

I have Pushed OoyalaAPI Sample App hook and raise a pull request. Please review the code and merge it with dev branch.

Let me know in case of more info needed or some changes are required.

### CODE REVIEW

File: OoyalaAPISampleApp/OoyalaAPISampleApp/AppDelegate.h [hunk 1]
```diff
@@ -11,6 +11,8 @@
 @interface AppDelegate : UIResponder <UIApplicationDelegate>
 
 @property (strong, nonatomic) UIWindow *window;
+//A counter for tracking the number of ooyala sdk events generated across all the test asset plays
+@property (nonatomic,assign) int count ;
```
Comment (brol1dev) [inline OoyalaAPISampleApp/OoyalaAPISampleApp/AppDelegate.h]: remove space between count and semicolon, should look like `count;`

File: OoyalaAPISampleApp/OoyalaAPISampleApp/Lists/DiscoveryListViewController.h [hunk 2]
```diff
@@ -11,5 +11,5 @@
 @interface DiscoveryListViewController : UITableViewController
 
 @property NSString *embedCode;
-
+@property BOOL qaModeEnabled;
```
Comment (brol1dev) [inline OoyalaAPISampleApp/OoyalaAPISampleApp/Lists/DiscoveryListViewController.h]: add nonatomic attribute

File: OoyalaAPISampleApp/OoyalaAPISampleApp/Lists/PlaylistListViewController.h [hunk 3]
```diff
@@ -5,4 +5,5 @@
 
 @property (nonatomic) PlayerSelectionOption *option;
 
+@property BOOL qaModeEnabled;
```
Comment (brol1dev) [inline OoyalaAPISampleApp/OoyalaAPISampleApp/Lists/PlaylistListViewController.h]: add nonatomic attribute

**Review by brol1dev — COMMENTED** [review]
You are not being consistent with the changes that we ask you to make on several PRs. We're asking the same thing over and over, be careful on when to use the appropriate attributes for properties, not adding additional spaces or not using spaces when required. Using different indentation spaces than what we already have in the code.

I'm letting some of them pass but this is starting to make the code look different than we already have. I know we need to improve what we already have in our apps, but let's not add more code that will need to be refactored in the future.

File: OoyalaAPISampleApp/OoyalaAPISampleApp/AppDelegate.h (hunk 1 above)
Comment (sgupta3112) [reply OoyalaAPISampleApp/OoyalaAPISampleApp/AppDelegate.h]: Done.

File: OoyalaAPISampleApp/OoyalaAPISampleApp/Lists/DiscoveryListViewController.h (hunk 2 above)
Comment (sgupta3112) [reply OoyalaAPISampleApp/OoyalaAPISampleApp/Lists/DiscoveryListViewController.h]: Done.

File: OoyalaAPISampleApp/OoyalaAPISampleApp/Lists/PlaylistListViewController.h (hunk 3 above)
Comment (sgupta3112) [reply OoyalaAPISampleApp/OoyalaAPISampleApp/Lists/PlaylistListViewController.h]: Done.

**Pull request merged**